src/models/db/types.ts

```diff
@@ -60,7 +60,7 @@ export type Models = {
 type WhereComparison<M extends TABLE> = {
   [key in keyof Models[M]['get']]: [
     Extract<key, string>,
-    '=' | '>' | '>=' | '<' | '<=' | '<>',
+    '=' | '>' | '>=' | '<' | '<=' | '<>' | 'LIKE' | 'ILIKE' | '~',
```

Comment:
let's remove `~` as we're not using it or planning on. I'm not sure the impact regular expresssions would have on things